pkg/controller/operconfig/ingressconfig_controller.go

```diff
@@ -63,15 +47,16 @@ func (r *ReconcileIngressConfigs) Reconcile(request reconcile.Request) (reconcil
 			log.Printf("Unable to retrieve IngressController.operator.openshift.io object: %v", err)
 			return reconcile.Result{}, err
 		}
-		labels := map[string]string{HostNetworkLabel: ""}
+		labels := map[string]*string{names.PolicyGroupLabel: nil}
 		log.Printf("Ingress Controller Config: %+v", ingressControllerConfig)
 		if ingressControllerConfig.Spec.EndpointPublishingStrategy != nil &&
 			ingressControllerConfig.Spec.EndpointPublishingStrategy.Type == operv1.HostNetworkStrategyType {
-			labels[HostNetworkLabel] = HostNetworkLabelValue
+			labelVal := names.PolicyGroupLabelIngressValue
+			labels[names.PolicyGroupLabel] = &labelVal
```

Comment:
I mean, I guess this works, but it's still just making extra work. There are only two possibilities here: either you want to set `names.PolicyGroupLabel`, or you want to clear `names.PolicyGroupLabel`. You don't need a fully generic `setLabelsOnNamespace` function that iterates over a map of changes. You just need to set or clear `names.PolicyGroupLabel`.